packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/sagemaker/base-types.ts

```diff
@@ -464,7 +464,12 @@ export enum InputMode {
   /**
    * File mode.
    */
-  FILE = 'File'
+  FILE = 'File',
+
+  /**
+   * FastFile mode.
+   */
+  FASTFILE = 'FastFile'
```

Comment:
Should be renamed `FAST_FILE` to follow the naming conventions in the rest of the file.

Reply:
OK, will change it